frontend/src/views/app-store/setting/index.vue

```diff
@@ -99,7 +103,18 @@ const setDefaultDomain = () => {
     });
 };
 
+const getNodeConfig = async () => {
+    if (globalStore.isMaster) {
+        return;
+    }
+    const res = await getCurrentNodeCustomAppConfig();
+    if (res && res.data) {
+        useCustomApp.value = res.data.status === 'enable';
+    }
+};
+
 onMounted(() => {
     search();
+    getNodeConfig();
 });
 </script>
```

Comment:
The code does not contain any clear violations of standards or potential issues. It doesn't seem to have significant design flaws in terms of layout, coding format or functionality. Any optimizations would mostly need context about how it's being used elsewhere in the app or system.

However, minor tweaks like removing extra space around `</div>`, improving naming conventions, checking that `<el-button>` always has an icon, etc., should be possible with minimal effort. If you're unsure what specific improvements could work best at this stage for the current task, I'd encourage you to review comments or guidelines provided by developers in similar projects.